test/FastTests/Sparrow/Hashing.cs

```diff
@@ -387,114 +284,250 @@ public unsafe void XXHash32_StreamedHashingEquivalence(int bufferSize)
             int iteration = 1;
             do
             {
-                blockSize = Hashing.Streamed.XXHash32.Alignment * iteration;
+                blockSize = Hashing.Streamed.XXHash64.Alignment * iteration;
 
-                var context = new Hashing.Streamed.XXHash32Context {Seed = seed};
-                Hashing.Streamed.XXHash32.BeginProcess(ref context);
+                var context = new Hashing.Streamed.XXHash64Context {Seed = seed};
+                Hashing.Streamed.XXHash64.Begin(ref context);
                 fixed (byte* buffer = values)
                 {
                     byte* current = buffer;
                     byte* bEnd = buffer + bufferSize;
                     do
                     {
                         int block = Math.Min(blockSize, (int)(bEnd - current));
-                        Hashing.Streamed.XXHash32.Process(ref context, current, block);
+                        Hashing.Streamed.XXHash64.Process(ref context, current, block);
                         current += block;
                     }
                     while (current < bEnd);
                 }
 
                 iteration++;
 
-                var result = Hashing.Streamed.XXHash32.EndProcess(ref context);
-                var expected = Hashing.XXHash32.Calculate(values, -1, seed);
+                var result = Hashing.Streamed.XXHash64.End(ref context);
+                var expected = Hashing.XXHash64.Calculate(values, -1, seed);
 
                 Assert.Equal(expected, result);
             }
             while (blockSize <= bufferSize);
         }
 
-        [Theory]
-        [MemberData("BufferSize")]
-        public unsafe void XXHash64_StreamedHashingEquivalence(int bufferSize)
+        [Fact]
+        public unsafe void XXHash64_HashingEquivalenceWithReference()
         {
             var rnd = new Random(1000);
```

Comment:
Can we pass a seed here? You can get it from [InlineDataWithRandomSeed]

Reply:
Done in all of them.